shell/browser/electron_autofill_driver.cc

```diff
@@ -55,9 +60,23 @@ void AutofillDriver::ShowAutofillPopup(
     embedder_frame_host = embedder->web_contents()->GetPrimaryMainFrame();
   }
 
+  // Ensure that if the WebContents belongs to a BrowserView,
+  // the popup is positioned relative to the BrowserView's bounds.
+  for (NativeBrowserView* bv : owner_window->browser_views()) {
```

Comment:
Would it make sense to make `NativeBrowserView` attach to the lifetime of `WebContents` via `WebContentsUserData`, that way querying the view from the contents would be straightforward.

Reply:
@deepak1556 does that work here for one-to-many relationships? my understanding (and all the examples I see in the codebase) are one-to-one

Reply:
done!